Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN yum -y install \
 
 EXPOSE 80
 
+RUN mkdir -p /var/lib/php/session
```

Comment:
Wouldn't that be a project/environment level decision to map a volume to this particular path?